src/scripts.js

```diff
@@ -356,7 +362,9 @@ function displayIngredientsNeeded(recipe) {
   const neededIngredientsSection = document.querySelector('.ingredients-needed-js');
 
   if (neededIngredients.length === 0) {
-    neededIngredientsSection.innerText = 'you have all of the ingredients needed to cook this recipe!';
+    neededIngredientsSection.innerHTML = `
+      <p>you have all of the ingredients needed to cook this recipe!</p>
+      <button>cook recipe</button>`;
```

Comment:
Decided to add the button that will remove ingredients from the user's pantry (api) in this area because it only shows up when user has enough ingredients in their pantry to cook this specific recipe.

Reply:
This is a great UI choice, should not render the button if they don't have enough to cook!